### PR TITLE
Chrisk bugfixes

### PR DESCRIPTION
**Pull request by chriskr**

### CODE REVIEW

File: src/ecma-debugger/jssourcetooltip.js [hunk 1]
```diff
@@ -384,6 +388,25 @@ cls.JSSourceTooltip = function(view)
...
 
+        if (shift_key && bracket_stack.length)
+        {
+          if (token[TYPE] == PUNCTUATOR)
+          {
+            if (token[VALUE] == "]")
+            {
+              bracket_stack.push(token[VALUE])
+              previous_token = token;
+            }
```
Comment (hzr) [inline src/ecma-debugger/jssourcetooltip.js:400]: Would like a newline after this.

File: src/ecma-debugger/jssourcetooltip.js [hunk 2]
```diff
@@ -590,6 +621,25 @@ cls.JSSourceTooltip = function(view)
...
 
+        if (shift_key && bracket_stack.length)
+        {
+          if (token[TYPE] == PUNCTUATOR)
+          {
+            if (token[VALUE] == "[")
+            {
+              bracket_stack.push(token[VALUE])
+              previous_token = token;
+            }
```
Comment (hzr) [inline src/ecma-debugger/jssourcetooltip.js:635]: Would like a newline after this.

**Comment by hzr** [comment]
Two comments, looks fine otherwise.

**Comment by hzr** [comment]
Line 634 also needs a newline.

**Comment by hzr** [comment]
The last commits look fine too. Like the idea of a fish as a dragonfly debugger!

**Comment by hzr** [comment]
Looks fine.

**Comment by danfooo** [comment]
The rest looks good.

**Comment by chriskr** [comment]
Improved two fixes.

**Comment by chriskr** [comment]
@hzr @danfoo one more fix for review

**Comment by hzr** [comment]
Looks fine!

**Comment by danfooo** [comment]
Yes, looks fine!

**Pull request merged**